conf/app.routes

```diff
@@ -1,7 +1,9 @@
 # microservice specific routes
 
-GET        /hello-world                                                 uk.gov.hmrc.debttransformationstub.controllers.MicroserviceHelloWorldController.hello
 GET        /debt                                                        uk.gov.hmrc.debttransformationstub.controllers.CaseManagementSystemController.getList
 GET        /debt/:debtID                                                uk.gov.hmrc.debttransformationstub.controllers.CaseManagementSystemController.getCaseDetails(debtID: String, duties: Option[String])
```

Comment:
debtID change to debtId in the sol service?

Reply:
Tbh this endpoint won't be used, I will remove once we pass testing